README.md

```
@@ -1,112 +1 @@
-[license-shield]: https://img.shields.io/github/license/quran/quran.com-frontend-next?style=for-the-badge
-[license-url]: https://github.com/quran/quran.com-frontend-next/blob/master/LICENSE.txt
-[product-screenshot]: images/screenshot.png
-
-### Credits
-
-- Localization was made possible by the help of [Lokalise](https://lokalise.com/) which is a computer-aided translation system that focuses on productivity and quality assurance and provides a seamless localization workflow.
-
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/15169499/139687128-15ed6189-6be2-44bf-9173-75cce317d546.png" width="400">
-</p>
+This is a test
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)